jboss-kie-kieserver/added/openshift-launch.sh

```diff
@@ -33,6 +33,7 @@ CONFIGURE_SCRIPTS=(
   $JBOSS_HOME/bin/launch/filters.sh
   $JBOSS_HOME/bin/launch/jboss-kie-wildfly-common.sh
   $JBOSS_HOME/bin/launch/jboss-kie-kieserver.sh
+  $JBOSS_HOME/bin/launch/jboss-kie-kieserver-jms-setup.sh
```

Comment:
I would prefer this script to be called "jboss-kie-kieserver-jms.sh" instead of "jboss-kie-kieserver-jms-setup.sh", since the "setup" part is implied. ALL our config scripts do setup, so it's redundant.

Reply:
step done.